packages/cozy-mespapiers-lib/src/components/Home/Home.spec.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
-import { render, fireEvent } from '@testing-library/react'
+import { render, fireEvent, waitFor } from '@testing-library/react'
+import '@testing-library/jest-dom'
```

Comment:
why this change? I don't think that is normal to add this line. Let's have the library in devDep.

Reply:
En effet, ce n'est pas utile.
Pour la raison, durant l'écriture des tests, j'ai voulu utiliser la méthode `toHaveAttribute`, et sans cette import, cela ne marche pas.